### DISCUSSION TITLE
Fix points on connection

### DISCUSSION THREAD

**Discussion by KMR1324** [freeplane/freeplane#730] [Help]
When creating a link/connection, I can't change it well because the points to adjust are not fixed and disappear as soon as I release the left mouse button. What are the correct settings to fix the points?

**Comment by euu2021** [comment]
Which version of FreePlane are you using?

The second image [here](https://docs.freeplane.org/attic/new-wiki-initiative/understanding-nodes.html?highlight=connec#connectors) is a gif showing how it works. See if it helps.

I think that, on latest versions of FreePlane, the system changed a bit, though.

**Comment by KMR1324** [comment]
Hi!

Thank you very much for your answer.

I have version freeplane 1.10.4.

Must i do an update for having the described functions about connectors?

Best wishes

Katrin

Von: euu ***@***.***>
Gesendet: Samstag, 8. Oktober 2022 17:23
An: freeplane/freeplane ***@***.***>
Cc: Katrin Müller-Riemenschneider ***@***.***>; Author ***@***.***>
Betreff: Re: [freeplane/freeplane] Fix points on connection (Discussion #730)


Which version of FreePlane are you using?

The second image here<https://docs.freeplane.org/attic/new-wiki-initiative/understanding-nodes.html?highlight=connec#connectors> is a gif showing how it works. See if it helps.

I think that, on latest versions of FreePlane, the system changed a bit, though.

—
Reply to this email directly, view it on GitHub<https://github.com/freeplane/freeplane/discussions/730#discussioncomment-3829241>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3PCPABVEPHEOIZ7MAPEY2LWCGGUHANCNFSM6AAAAAAQ7OA3RI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/freeplane/freeplane/discussions/730#discussioncomment-3829241", "url": "https://github.com/freeplane/freeplane/discussions/730#discussioncomment-3829241", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by euu2021** [reply]
No, you don't need to update. I mentioned that the latest versions were different, but I think I was wrong. I guess I confused it with another feature that was changed.
